src/components/datepicker/js/calendar.spec.js

```diff
@@ -113,7 +113,14 @@ describe('md-calendar', function() {
   function dispatchKeyEvent(keyCode, opt_modifiers) {
     var mod = opt_modifiers || {};
 
-    angular.element(document.body).triggerHandler({
+    var dispatchElement;
+      if (angular.element(element).parent().hasClass('md-datepicker-calendar')) {
```

Comment:
You don't need this check. The calendar here will always be on it's own (without the datepicker).